### PR TITLE
Better dependencies declaration

### PR DESCRIPTION
**Pull request by artem-zinnatullin**
Little nice thing.

@nikitin-da PTAL

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -34,36 +34,47 @@ ext {
...
 }
 
+ext.libraries = [
+        // Core libraries
+        storIOCommon                   : project(':storio-common'),
+        supportAnnotations             : 'com.android.support:support-annotations:22.2.0',
+        rxJava                         : 'io.reactivex:rxjava:1.0.11',
+
+        // Libraries for tests and sample app
+        storIOSQLite                   : project(':storio-sqlite'),
```
Comment (nikitin-da) [inline build.gradle]: Are `storio-sqlite` and etc libraries for test or for sample app?

Comment (nikitin-da) [inline build.gradle]: I think it's core too

Comment (artem-zinnatullin) [inline build.gradle]: Core libraries — libraries that needed for `StorIOSQLite` and `StorIOContentResolver` :)

So, I think `StorIOSQLite` and `StorIOContentResolver` are not core libraries.

Comment (nikitin-da) [inline build.gradle]: Yes, call that modules as `core` is not entirely accurate. But I don't agree that it's just for test or for sample app =)

Comment (artem-zinnatullin) [inline build.gradle]: Hehe :) So, how should we call them?

Comment (nikitin-da) [inline build.gradle]: I all these modules in block, maybe `facilities`?

Comment (artem-zinnatullin) [inline build.gradle]: Noooo :) Sounds bad

I think we can make block `// StorIO parts`

Comment (nikitin-da) [inline build.gradle]: :+1:

**Pull request merged**